Source/Projections/Store/IProjections.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Dolittle.SDK.Projections.Store
+{
+    /// <summary>
+    /// Defines an interface for working directly with Projections.
+    /// </summary>
+    public interface IProjections
```

Comment:
Should we also rename this to be consistent?
```suggestion
    public interface IProjectionStore
```